deploy/runtools/firesim_topology_with_passes.py

```diff
@@ -294,6 +296,7 @@ def pass_apply_default_network_params(self):
                     node.server_link_latency = self.defaultlinklatency
                 if node.server_bw_max is None:
                     node.server_bw_max = self.defaultnetbandwidth
+                # TODO: some of this stuff seems misplaced...
```

Comment:
What do you mean? It shouldn't be with the network stuff?

Reply:
yeah. it doesn't really matter, but maybe this function should just be renamed.